### PR TITLE
Added output flag for CLI

### PR DESCRIPTION
**Pull request by jonnyarnold**
Now we can specify a filename to output the result to!

### CODE REVIEW

**Comment by reevoo-samuel** [comment]
_If you're new, read the [guidelines](https://github.com/reevoo/guidelines/blob/master/pull_requests.md)._

Good code is **tested**, **easy to understand** and **able to cope when things go wrong**.

Good pull requests are **descriptive**, **small** and **short-lived**.

**For the Author**
- [x] Description explains what you have done and why.
- [ ] Tests have been written (or you have explained why you don't need tests).
- [x] Documentation/comments have been added/updated.

**For the Reviewer**
- [ ] Tests are green on CI (or pulled/tested on your machine).
- [ ] Tests cover happy and unhappy paths in the code.
- [ ] The code is easy to understand.

**Comment by mikekr1** [comment]
Damn son, where your tests at?

**Comment by jonnyarnold** [comment]
It's next on my list https://github.com/reevoo/gem-miner/issues/4

The shame...

File: lib/gem-miner/cli.rb [hunk 1]
```diff
@@ -27,7 +32,21 @@ class CLI < Thor
...
-      puts GemMiner::Miner.gems_for(github_search_query, options[:token]).to_yaml
+      output GemMiner::Miner.gems_for(github_search_query, options[:token]).to_yaml
+    end
+
+    private
+
+    # Output the result of a command to either the terminal or a file,
+    # depending on the options given.
+    def output(result)
+      unless options[:output]
```
Comment (Antti) [inline lib/gem-miner/cli.rb]:
``` ruby
if options[:output]
else
end
```

Maybe?

Comment (jonnyarnold) [inline lib/gem-miner/cli.rb]: So the only reason for that was that the top option is the most likely.

**Comment by jonnyarnold** [comment]
LGTM @Antti ?

File: lib/gem-miner/cli.rb (hunk 1 above)
Comment (Antti) [inline lib/gem-miner/cli.rb]: Shouldn't matter. It's easier to read `if  x ... else ...`,  rather then `unless y ... else`

**Pull request merged**